docs/proposal.md

```diff
@@ -10,8 +10,23 @@ This "movies" dataset is from the [Vega Datasets][1] collection.
 TBC
 
 ## Research Questions and Usage Scenarios
+### Research questions:
+
+1. How does the US gross revenue different by movie genre?
+
+2. What is the relationship between rating and duration of the movie?
+
+3. What are the top 10 rated movies at the year(s) of interest by genre? 
```

Comment:
```suggestion
3. What are the top 10 rated movies at the year(s) of interest (by genre)? 
```
As per discussed in the Slack with Junrong, we think it's better to include () as genre is optional